theano/ifelse.py

```diff
@@ -245,12 +246,14 @@ def thunk():
                     else:
                         for out, outtype, t in izip(outputs, outtypes, ts):
```

Comment:
You don't need the outtype anymore, so maybe you can remove it from the iteration.